projects/assets-library/src/icons/icon-type.ts

```diff
@@ -99,6 +99,7 @@ export const enum IconType {
   Share = 'share',
   SidebarCollapse = 'svg:sidebar-collapse',
   SidebarExpand = 'svg:sidebar-expand',
+  SidebarLeft = 'svg:sidebar-left',
```

Comment:
@palbizu I don't see an svg for this in the assets. Did you already add it?

Reply:
fixed. My mistake, I added in the wrong folder.